src/main/java/org/kiwiproject/dropwizard/util/health/MongoHealthCheck.java

```diff
@@ -28,28 +28,29 @@ public MongoHealthCheck(DB db) {
     @SuppressWarnings({"deprecation", "ConstantConditions"})
     @Override
     protected Result check() throws Exception {
-        try {
-            var host = db.getMongo().getAddress().toString();
-            var dbName = db.getName();
+        var host = db.getMongo().getAddress().toString();
```

Comment:
Just read Mongo code...`getAddress()` could return `null` so we should handle that rather than let the NPE be thrown...

Reply:
...and the address _could_ change, for example if the current master (in a cluster) changes.

Reply:
Throw `IllegalStateException`?

Reply:
No sorry, it's not a problem. My idea to store the host and database name in fields doesn't work (at least for the host) since it could change due to the cluster master changing. So this is all good. Sorry for the confusion.